src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -82,8 +113,40 @@ export const IncidentMap = () => {
           attributionControl: false,
         }}
       >
-        <IncidentLayer passBbox={setBbox} incidents={filteredIncidents} />
-        {map && <GPSLocation map={map} setNotification={setNotification} />}
+        <IncidentLayer
+          passBbox={setBbox}
+          incidents={filteredIncidents}
+          handleIncidentSelect={handleIncidentSelect}
+          handleCloseDetailPanel={handleCloseDetailPanel}
+          resetMarkerIcons={resetMarkerIcons}
+        />
+
+        {map && (
+          <GPSLocation
+            map={map}
+            setNotification={setNotification}
+            panelIsOpen={drawerState}
+          />
+        )}
+
+        <DrawerOverlay
+          onStateChange={setDrawerState}
+          state={drawerState}
+          ControlledContent={(props) => (
+            // TODO: replace this with the actual Address Search (to be build)
```

Comment:
Is there a story to create the actual address search? in that case remove the todo (less pollution)

Reply:
@SireeKoolenWijkstra is working on it. This is the [Jira Ticket](https://datapunt.atlassian.net/browse/SIG-4784). Removed the dummy code, but did leave a comment so er knew the intention of this piece.